swift-swiftLint.md

```diff
@@ -0,0 +1,131 @@
+# Introduction
+
+[SwiftLint](https://github.com/realm/SwiftLint/blob/master/README.md) is a tool by Realm to enforce Swift style and conventions. SwiftLint hooks into Apple’s own SourceKit framework to parse your Swift code, which means it supports the full range of Swift syntax and remains up to date as Swift continues to evolve.
+<br/>It is based on guidelines from the Swift style guide.
```

Comment:
I would skip the `<br/>` tags and instead add just empty lines.

Reply:
updated